client/app/components/Modal/Modal.scss

```diff
@@ -33,8 +33,10 @@
     height: 60vh;
     z-index: $z-index-front;
     @media screen and (max-width: $small) {
+      position: absolute;
```

Comment:
Actually, I just saw this... Is this necessary?

Reply:
I think it was there to begin with. I can check to see if it's actually necessary or not right now.

Reply:
Looks like it's been added. Yeah, let's check if that and `margin: 0 auto` is necessary! If not let's remove it!

Reply:
I removed both margin: 0 auto and position: absolute. It works without those. I must have slid them in thinking they would be useful when they aren't.